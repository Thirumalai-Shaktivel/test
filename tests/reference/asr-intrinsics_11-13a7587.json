{
    "basename": "asr-intrinsics_11-13a7587",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/intrinsics_11.f90",
    "infile_hash": "22ac1dd1527a39d35236a50e3b1afe37ab40b0970f22d259a077800f",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-intrinsics_11-13a7587.stdout",
    "stdout_hash": "67f09a25e4d404c809f0309c5e4d4a9f536cfd426b5b9ddcea14663d",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}