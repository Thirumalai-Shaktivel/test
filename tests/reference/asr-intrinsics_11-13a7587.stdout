(TranslationUnit (SymbolTable 1 {intrinsics_11: (Program (SymbolTable 2 {real: (ExternalSymbol 2 real 4 real lfortran_intrinsic_array real Private), x: (Variable 2 x Local (ConstantInteger 3 (Integer 4 [])) (ConstantInteger 3 (Integer 4 [])) Parameter (Integer 4 []) Source Public Required), y: (Variable 2 y Local (FunctionCall 2 real () [(Var 2 x)] [] (Real 4 []) (ConstantReal 3.000000 (Integer 4 [])) ()) (ConstantReal 3.000000 (Integer 4 [])) Parameter (Real 4 []) Source Public Required)}) intrinsics_11 [lfortran_intrinsic_array] [(Print () [(Var 2 y)])]), lfortran_intrinsic_array: (IntrinsicModule lfortran_intrinsic_array)}) [])
